### ISSUE TITLE
Collision of mibs names

### ISSUE THREAD

**Issue by mbn18** [sys4/pacemaker-snmp#2]
On CentOS/RHEL 6.5 the package **pacemaker-1.1.10-14.el6_5.3** provide the following mib which use the name **PACEMAKER-MIB** :

```
# head /usr/share/snmp/mibs/PCMK-MIB.txt
PACEMAKER-MIB DEFINITIONS ::= BEGIN

--
-- MIB objects for the pacemaker cluster manager implementation
--

IMPORTS
    MODULE-IDENTITY, OBJECT-TYPE, Integer32,
    NOTIFICATION-TYPE, enterprises          FROM SNMPv2-SMI
    SnmpAdminString                         FROM SNMP-FRAMEWORK-MIB
```

which is identical to the one at:

```
# head PACEMAKER.mib
PACEMAKER-MIB DEFINITIONS ::= BEGIN

-- SYS4 AG
-- Franziskaner Str. 15
-- 81669 Munich
-- Germany

-- SMIv2
-- December 27th, 2013.

```

**Comment by misch42** [comment]
Ah. thanks for reporting it. I will check.

Michael.

**Comment by misch42** [comment]
Hi,

solved the issue with 839e630.

It was a stupid error since I also wrote the PACEMAKER-MIB which contains the notifications sent out by the cluster resource manager itself.
